### PR TITLE
Update documentation product video configuration

### PR DESCRIPTION
**Pull request by rodrini-the-po**
The existing was not clear on how to collect the key from the Google cloud console. All steps have been updated.

## Purpose of this pull request

This pull request (PR) updates the Product video documentation, since the Google Cloud console instructions are not clear and wrong in some instances. 

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/catalog/product-video.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by leslietilling** [comment]
@rodrini-the-po  Does this fix apply to both 2.3 and 2.4 docs?

File: src/catalog/product-video.md [hunk 1]
```diff
@@ -8,15 +8,17 @@ To add product video, you must first obtain an API Key from your Google account
 
 1. Log in to your Google account and visit the [Google Developers Console][1].
 
-1. Under _Use Google APIs_, click **Enable and manage APIs**.
+1. Search _YouTube Data API v3_ in the Search field above
```
Comment (leslietilling) [inline src/catalog/product-video.md:11]:
```suggestion
1. In the Search field at the top, enter `YouTube Data API v3` and click the search icon.
```

File: src/catalog/product-video.md [hunk 2]
```diff
@@ -8,15 +8,17 @@ To add product video, you must first obtain an API Key from your Google account
...
+
+1. Make sure the API has been enabled
```
Comment (leslietilling) [inline src/catalog/product-video.md:13]:
```suggestion
1. When the API page is displayed, make sure it is enabled.
```

File: src/catalog/product-video.md [hunk 3]
```diff
@@ -8,15 +8,17 @@ To add product video, you must first obtain an API Key from your Google account
...
 
 1. In the left panel, choose **Credentials**.
 
-1. Expand the _Add Credentials_ menu, and choose **API key**.
+1. Either copy the Key in the _API keys_ table or click on _Create Credentials_ on top and select **API key**.
```
Comment (leslietilling) [inline src/catalog/product-video.md:17]:
```suggestion
1. Depending on whether you have credentials or not, do one of the following:

   - If you already have the needed credentials, copy the key in the _API keys_ table.

   - If you do not already have credentials for this API, click <span class="btn">Create Credentials</span>  at the top and follow the prompts to create the needed credentials. Under _Get your credentials_, copy the API key and click <span class="btn">Done</span>.
```

File: src/catalog/product-video.md [hunk 4]
```diff
@@ -8,15 +8,17 @@ To add product video, you must first obtain an API Key from your Google account
...
 
-1. When prompted to create a new key, choose **Server key**.
+1. Copy the API key to the clipboard.
```
Comment (leslietilling) [inline src/catalog/product-video.md:19]: This step can be eliminated since the previous one already copies the key.

**Comment by rodrini-the-po** [comment]
Both

File: src/catalog/product-video.md [hunk 5]
```diff
@@ -8,15 +8,17 @@ To add product video, you must first obtain an API Key from your Google account
...
 
-1. Enter a name for the key and click <span class="btn">Create</span>.
+1. Click on the _edit_ icon on the right and make sure the API key is limited to the correct referers.
```
Comment (leslietilling) [inline src/catalog/product-video.md:21]:
```suggestion
1. Click the Edit icon on the right and set the restrictions to make sure the API key is limited to the correct referrers.
```

File: src/catalog/product-video.md [hunk 6]
```diff
@@ -50,6 +52,8 @@ To add product video, you must first obtain an API Key from your Google account
 
 1. Enter the **URL** of the YouTube or Vimeo video.
 
+1. Click outside the field and wait for feedback on the API key or video. If everything checks out, Youtube provides base information of the video
```
Comment (leslietilling) [inline src/catalog/product-video.md:55]: We separate the result or explaintion of the step as a nested paragraph.

```suggestion
1. Click outside the field and wait for feedback on the API key or video.

   If everything checks out, Youtube provides base information of the video
```

**Comment by rodrini-the-po** [comment]
@leslietilling , updated doc with fixes on markdown

**Comment by leslietilling** [comment]
running tests

**Pull request merged**